docs/create-sharded-table.rst

```diff
@@ -0,0 +1,120 @@
+.. _create-sharded-table:
+
+====================
+Create sharded table
+====================
+
+One core concept CrateDB uses to distribute data across a cluster is sharding. 
+CrateDB splits every table into a configured number of shards, which are 
+distributed evenly across the cluster. You can think of shards as sub-tables 
+with corresponding sub-indexes. If we create a table like the following:
+
+.. code-block:: psql
+
+    CREATE TABLE first_table (
+        ts TIMESTAMP,
+        val DOUBLE PRECISION
+    );
+
+The table is by default is split into 4 shards on a single node cluster. You 
+can check this by running:
+
+.. code-block:: psql
+
+    SHOW CREATE TABLE first_table;
+
+Which should output the following:
+
+.. code-block:: psql
+
+    CREATE TABLE IF NOT EXISTS "doc"."first_table" (
+        "ts" TIMESTAMP WITH TIME ZONE,
+        "val" DOUBLE PRECISION
+    )
+    CLUSTERED INTO 4 SHARDS
+
+By default, ingested data is distributed evenly across all available shards. In 
+many cases you don't have to worry about specifying any sharding key or routing 
+column. 
```

Comment:
It's not yet mentioned that users can specify a routing column, so this might surprise and confuse readers. 
Maybe it's better to mention that they could specify a routing column to have more control over the data distribution, but that it's usually not necessary and add a link to the CREATE TABLE statement reference?